ingest/scripts/call_loculus.py

```diff
@@ -173,52 +173,20 @@ def get_or_create_group_and_return_group_id(config: Config, allow_creation: bool
     return create_group_and_return_group_id(config)
 
 
-def filter_fasta_by_submission_ids(
-    fasta_dict: dict[str, Any], submission_ids: list[str], config: Config
-):
-    if config.segmented:
-        segmented_submission_ids = [
-            f"{submission_id}_{segment}"
-            for submission_id in submission_ids
-            for segment in config.nucleotide_sequences
-        ]
-        return [
-            fasta_dict[submission_id]
-            for submission_id in segmented_submission_ids
-            if submission_id in fasta_dict
-        ]
-    return [
-        fasta_dict[submission_id] for submission_id in submission_ids if submission_id in fasta_dict
-    ]
-
-
 def post_fasta_batches(
     url,
     fasta_file: str,
     metadata_file: str,
     config: Config,
     params: dict[str, str],
-    chunk_size=60000,
+    chunk_size=1000,
```

Comment:
I decreased this to see it in action on the preview - it can be increased before submission
```suggestion
    chunk_size=10000,
```